Source/Addon/Custom Device Instrument Addon.xml

```diff
@@ -57,63 +57,63 @@
       <SupportedTarget>Windows</SupportedTarget>
       <Source>
         <Type>To Common Doc Dir</Type>
-        <Path>Custom Devices\Instrument Addon\Windows\Protocols.lvlibp</Path>
+        <Path>Custom Devices\National Instruments\Instrument Addon\Windows\Protocols.lvlibp</Path>
       </Source>
       <RealTimeSystemDestination>c:\ni-rt\NIVeriStand\Custom Devices\Instrument Addon\Protocols.lvlibp</RealTimeSystemDestination>
     </Dependency>
     <Dependency>
       <SupportedTarget>Windows</SupportedTarget>
       <Source>
         <Type>To Common Doc Dir</Type>
-        <Path>Custom Devices\Instrument Addon\Windows\Protections.lvlibp</Path>
+        <Path>Custom Devices\National Instruments\Instrument Addon\Windows\Protections.lvlibp</Path>
       </Source>
       <RealTimeSystemDestination>c:\ni-rt\NIVeriStand\Custom Devices\Instrument Addon\Protections.lvlibp</RealTimeSystemDestination>
     </Dependency>
       <Dependency>
       <SupportedTarget>Pharlap</SupportedTarget>
       <Source>
         <Type>To Common Doc Dir</Type>
-        <Path>Custom Devices\Instrument Addon\Pharlap\Protocols.lvlibp</Path>
+        <Path>Custom Devices\National Instruments\Instrument Addon\Pharlap\Protocols.lvlibp</Path>
```

Comment:
I'm worried these path changes will require mutation, since the driver VI paths are embedded into system definitions. Please validate that system definitions created with previous versions of this custom device continue to deploy after these paths change.